dds/DCPS/LinuxNetworkConfigMonitor.cpp

```diff
@@ -38,7 +38,14 @@ bool LinuxNetworkConfigMonitor::open()
   ACE_Netlink_Addr addr;
   addr.set(pid, RTMGRP_NOTIFY | RTMGRP_LINK | RTMGRP_IPV4_IFADDR | RTMGRP_IPV6_IFADDR);
   if (socket_.open(addr, AF_NETLINK, NETLINK_ROUTE) != 0) {
+#ifdef ACE_ANDROID
+    if (DCPS_debug_level >= 1) {
+      ACE_DEBUG((LM_WARNING, ACE_TEXT("(%P|%t) WARNING: LinuxNetworkConfigMonitor::open: could not open socket"
+        " this is expected for API30+\n")));
```

Comment:
`ACE_TEXT` can't span multiple strings since all it does is expand to `L` or nothing to change if the literal is wide or narrow. Either add another `ACE_TEXT` to the second literal or remove it entirely. Personally I've been leaving them off for a while.